po-refresh

```diff
@@ -140,6 +140,11 @@ def run(context, verbose=False, **kwargs):
                 current_linguas.remove(locale[0])
             (user, local_branch) = add_and_commit_lang(name, locale[0], "Drop").split(":")
 
+    # HACK: Semicolon in some languages in 'Plural-Forms' header key can break translations
+    # https://github.com/mikeedwards/po2json/issues/87
+    # Removing all semicolons is safe way around this
+    subprocess.run(r'sed -i "s/\(^.Plural-Forms:.*\);\\\n\"$/\1\\\n\"/" po/*.po', shell=True)
```

Comment:
Suggestion to make this a bit simpler: Call sed with single quotes, and without shell, so that you don't need so much escaping; use a separate match and replace pattern; and use `-r` so that you don't need to escape the ():

     subprocess.check_call(['sed', '-ri', r'/^"Plural-Forms:/ s/;(\\n")$/\1/'] + glob.glob("po/*.po"))